README.md

```diff
@@ -1,3 +1,62 @@
 # App Store Analytics Export
 
-Scripts for exporting app analytics to BigQuery
+Scripts for exporting app analytics to BigQuery.
+The export will export individual metrics grouped by a single dimension and load each
+metric and dimension combination into a single table.
+Metrics and dimensions that will be exported can be found in the 
+[metric metadata file](analytics_export/tableMetadata.js).
+
+This export uses the 
+[iTunesConnectAnalytics](https://github.com/JanHalozan/iTunesConnectAnalytics) package to handle authentication.
+A username and password for App Store Connect is required to use the script. 
+
+## Usage
+
+This project uses `yarn` to manage dependencies and was developed with nodejs 12.
+
+To install dependencies use:
+```sh
+yarn
+```
+
+To run the export use `yarn export` which takes the following arguments:
+```sh
```

Comment:
This may (will?) go out of date. I'd recommend just suggesting running `yarn export --help` to see all the options. The concrete example below is good though, keep that. :)